.devcontainer/Dockerfile

```diff
@@ -19,3 +19,6 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
 
 # Install LLVM
 RUN bash -c "cd /tmp && wget https://apt.llvm.org/llvm.sh && chmod +x ./llvm.sh && ./llvm.sh 14"
+
+ADD ../src /src
```

Comment:
I think we should make the entire repository available inside the container, as we're not concerned with the size of the resulting image.

Another (opinionated) point I was trying to make earlier is that the devcontainer should mount files, not have copies inside the image. This is because it's for development. If you copy the files, your changes on local files won't be available inside the container, whereas if you mount them, you can change files and run them in the container.

```bash
cd .devcontainer
docker build . -t tivo-devcontainer
cd .. # go back to project root
docker run --rm -v $(pwd):/workspaces/tivo -it -w /workspaces/tivo tivo-devcontainer
```

I'm imagining this to be used like this, and any tools that follows the [development container](https://containers.dev) (like VSCode and GitHub Codespaces) do this automatically.

Reply:
I couldn't get `docker run --rm -v $(pwd):/workspaces/tivo -it -w /workspaces/tivo tivo-devcontainer` working locally but am cool to just follow the VSCode approach